app/components/mandatarissen/mandataris-card.hbs

```diff
@@ -103,24 +103,39 @@
         </div>
       </div>
     {{/if}}
-    {{#if (await this.vervangersDoor)}}
-      <div class="au-o-grid__item au-u-5-6">
+    {{#if (await @mandataris.status.isVerhinderd)}}
+      <div class="au-o-grid__item au-u-1-2">
+        <div class="au-c-description-label"></div>
+        <div class="au-c-description-value au-u-margin-top-tiny">
+        </div>
+      </div>
+      <div class="au-o-grid__item au-u-1-2">
         <div class="au-c-description-label"><div>Vervangen door:</div></div>
-        <div class="au-c-description-value">
-          {{#each (await this.vervangersDoor) as |replacement index|}}
-            <AuLink
-              @route="mandatarissen.mandataris"
-              @model={{replacement.id}}
-              @skin="secondary"
-            >
-              {{replacement.isBestuurlijkeAliasVan.gebruikteVoornaam}}
-              {{replacement.isBestuurlijkeAliasVan.achternaam}}{{if
-                (lt index (sub this.vervangersDoor.length 1))
-                ","
-              }}
-            </AuLink>
-
-          {{/each}}
+        <div class="au-c-description-value replacement">
+          {{#if (not (await this.vervangersDoor))}}
+            <Mandatarissen::ReplacementSelect
+              @showDelete={{false}}
+              @showEditAsIcon={{true}}
+              @nothingSelectedText="Geen vervanger geselecteerd, voeg er een toe door op het potloodje te klikken."
```

Comment:
Do we really need the "voeg er een toe door op het potloodje te klikken"? Feels a bit over the top.

Reply:
Yes Saska already mentioned this in the Jira ticket but dind't update it it yet :D

Reply:
"Voeg vervanger toe"